openlibrary/templates/work_search.html

```diff
@@ -57,7 +57,10 @@ <h1>$_("Search Books")</h1>
         $# results because of performance issues due to increased load. See
         $# this commit to revert.
         <center>
-          <div class="red">$:_('No <strong>%(path_id)s</strong> directly matched your search', path_id=path_id)</div>
+          <div class="red">
+            $:_('No <strong>%(path_id)s</strong> directly matched your search.', path_id=path_id)
+            <a href="/books/add" class="btn btn-primary btn-sm ml-2">$_('Add a new book?')</a>
```

Comment:
We don't use tailwind

```suggestion
            <a href="/books/add">$_('Add a new book?')</a>
```